lodash.js

```diff
@@ -9000,14 +9000,12 @@
      * jQuery(window).on('popstate', debounced.cancel);
```

Comment:
The debounce code can be a bit complex and understanding the ins and outs of its async-ness can be challenging at times. Would you mind walking me through the changes step by step (baby step it)?